nucypher/cli/commands/worklock.py

```diff
@@ -115,7 +115,7 @@ def status(general_config, registry_options, worklock_options):
 @group_worklock_options
 @option_force
 @option_hw_wallet
-@click.option('--value', help="ETH value of bid", type=click.STRING)
+@click.option('--value', help="ETH value of bid", type=click.FloatRange(min=0))
```

Comment:
Can this min. be based on economics?

Reply:
I have no idea how to inject token_economics here, because it will be available only after reading from blockchain

Reply:
Hah, right... perhaps we can set the type inside the scope of the function, lazily.